CHANGELOG.md

```diff
@@ -7,8 +7,12 @@
         - Send text alerts for report updates to only-phone-verified users.
         - Add options for user to set global notification preferences.
         - Pop over mobile navigation menu. #3270
+<<<<<<< HEAD
         - Add support for the OS Maps API. #3328
         - Speed up /report/new/ajax call. #3335
+=======
+        - Improve `#geolocate_link` display, especially for smaller screens. #2048
+>>>>>>> Improve display of geolocation button.
```

Comment:
You've committed git conflict markers. Not sure what tool you use, but you should be able to have something that alerts you to this.

Reply:
Ok thanks I've removed git markers. I'm using VS Code - any plugins you can recommend for alerts?

Reply:
As I understand it, VS Code should be highlighting conflicts for you. When I do a merge with conflicts, I get something like this in the VS Code sidebar:
![image](https://user-images.githubusercontent.com/154364/110348950-1ab54080-802a-11eb-98c1-23f8e7f7d7d3.png)

Opening one of those files gives
![image](https://user-images.githubusercontent.com/154364/110349020-2e60a700-802a-11eb-8650-7b6697096467.png)

which I can resolve manually or using the helper buttons. If I try and commit using VS Code before resolving the conflicts, it gives me an error: "you have unmerged files". So it seems to do what you'd want already. So I guess the question is what are you doing when presented with a file with conflicts to resolve the conflict but leave the conflict markers in the file - are you somehow "git adding" it from the command line before resolving the conflict in the file?